src/main/java/org/sagebionetworks/bridge/hibernate/LabelConverter.java

```diff
@@ -0,0 +1,24 @@
+package org.sagebionetworks.bridge.hibernate;
+
+import javax.persistence.Converter;
+
+import org.sagebionetworks.bridge.models.Label;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+
+@Converter
+public class LabelConverter extends BaseJsonAttributeConverter<Label> {
```

Comment:
This class might not be necessary now that you're using the LabelListConverter.

Reply:
That's a good point. I checked with Shannon and I think you can keep it in there. Seems like it could get some use when more localization support is added in the future.